subject: Avoid readable stream
author: Bastien Roucariès <rouca@debian.org>

Avoid readable stream. Less code less error

Forwarded: not-needed

Index: node-fs-write-stream-atomic/index.js
===================================================================
--- node-fs-write-stream-atomic.orig/index.js
+++ node-fs-write-stream-atomic/index.js
@@ -1,5 +1,5 @@
 var fs = require('graceful-fs')
-var Writable = require('readable-stream').Writable
+var Writable = require('stream').Writable
 var util = require('util')
 var MurmurHash3 = require('imurmurhash')
 var iferr = require('iferr')
Index: node-fs-write-stream-atomic/package.json
===================================================================
--- node-fs-write-stream-atomic.orig/package.json
+++ node-fs-write-stream-atomic/package.json
@@ -9,8 +9,7 @@
   "dependencies": {
     "graceful-fs": "^4.1.2",
     "iferr": "^0.1.5",
-    "imurmurhash": "^0.1.4",
-    "readable-stream": "1 || 2"
+    "imurmurhash": "^0.1.4"
   },
   "devDependencies": {
     "rimraf": "^2.4.4",
